Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -668,27 +668,27 @@
         </trans-unit>
         <trans-unit id="ErrorDialogUnsupportedOperation" translate="yes" xml:space="preserve">
           <source>The requested operation is not supported</source>
-          <target state="translated">A kért művelet nem támogatott</target>
+          <target state="translated">Ez a művelet nem támogatott</target>
         </trans-unit>
         <trans-unit id="DriveFreeSpaceAndCapacity" translate="yes" xml:space="preserve">
           <source>{0} free of {1}</source>
-          <target state="translated">{0} szabad a(z) {1}-ból/ből</target>
```

Comment:
The main reason I've changed it was the "-ból/ből" suffix, what is not a really good concept, just like having "a(z)". I think we should try to avoid these as much as possible. This way the result looks better and easier to read.

Reply:
That's true. I just dont like the ''Szabad hely:". It's too preview window for me.

Reply:
What about "Elérhető:"?

Reply:
I have no problem with 'Szabad hely'. I just don't like the ':' in this case. It has a preview pane or details window feeling.

Reply:
If we can't come up with something better, it's a small price for readability.